CODEOWNERS

```diff
@@ -1,6 +1,4 @@
 # These datasets are subject to the additional change control procedures
 # described in https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/
-/sql/moz-fx-data-shared-prod/contextual_services @whd @jasonthomas
-/sql/moz-fx-data-shared-prod/contextual_services_derived @whd @jasonthomas
```

Comment:
We no longer need codeowners on `contextual_services*` paths because there are no longer any search terms under this path.